build.sbt

```
@@ -113,14 +94,32 @@ lazy val microservice = Project(appName, file("."))
     "-Wconf:cat=unused&src=.*ReverseRoutes\\.scala:s"
     )
   )
-lazy val ComponentTest = config("component") extend Test
-
-commands ++= Seq(
-  Command.command("run-all-tests") { state => "test" :: "it:test" :: "component:test" :: state },
 
-  Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
+lazy val it = (project in file("it"))
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(
+    name := "integration-tests",
+    Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-eT"),
+    DefaultBuildSettings.itSettings()
+  )
 
-  // Coverage does not need compile !
-  Command.command("pre-commit") { state => "clean" :: "scalafmtAll" :: "scalafixAll" :: "coverage" :: "testOnly * -- -l ExcludeFromCoverage" :: "it:test" :: "component:test" :: "coverageOff" :: "testOnly * -- -n ExcludeFromCoverage" :: "coverageReport" :: state }
+lazy val component = (project in file("component"))
+  .enablePlugins(PlayScala)
+  .dependsOn(microservice % "test->test")
+  .settings(
+    name := "component-tests",
+    libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test ++ AppDependencies.componentTestDependencies,
+    Test / testOptions := Seq(Tests.Argument(TestFrameworks.ScalaTest, "-eT")),
```

In other repos we have `+=` as in
```suggestion
    Test / testOptions += Seq(Tests.Argument(TestFrameworks.ScalaTest, "-eT")),
```

---

That's how I found it so haven't changed it.

---

Also you will note that in some places we set the value to a Seq whereas in others we add a value to the existing Seq.
I may changed them all to add to the Seq for consistency